src/services/users.service.ts

```diff
@@ -52,6 +52,17 @@ class UserService {
     const deleteUserData = await this.users.delete({ where: { id: userId }, select: { email: true, id: true } });
     return deleteUserData;
   }
+
+  public async manageGroups(userId: number, data: any) {
```

Comment:
`manageGroup` has data set to any it would be good to use an appropriate type. Also set the return value of `manageGroup` to be `promise<void>`.

psdt: we should avoid any whenever we can